addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -145,8 +149,16 @@ def _select(self) -> SQL:
                 END as is_unrated,
                 C.livechat_operator_id as partner_id,
                 CASE WHEN channel_member_history.has_agent THEN 1 ELSE 0 END as handled_by_agent,
-                CASE WHEN channel_member_history.has_bot and not channel_member_history.has_agent THEN 1 ELSE 0 END as handled_by_bot
-            """,
+                CASE WHEN channel_member_history.has_bot and not channel_member_history.has_agent THEN 1 ELSE 0 END as handled_by_bot,
+                CASE WHEN discuss_call_history.start_dt IS NOT NULL THEN 1 ELSE 0 END as has_call,
+                AVG(
```

Comment:
Shouldn't it be sum rather than average?
I'm thinking we want to now the whole call duration for a given session instead.